modules/adagioAnalyticsAdapter.js

```diff
@@ -5,16 +5,34 @@
 import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
 import adapterManager from '../src/adapterManager.js';
 import CONSTANTS from '../src/constants.json';
-import { getWindowTop } from '../src/utils.js';
+import { ajax } from '../src/ajax.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { getWindowTop, getWindowSelf, deepAccess, logInfo, logError } from '../src/utils.js';
+import { getGlobal } from '../src/prebidGlobal.js';
 
 const emptyUrl = '';
 const analyticsType = 'endpoint';
 const events = Object.keys(CONSTANTS.EVENTS).map(key => CONSTANTS.EVENTS[key]);
 const VERSION = '2.0.0';
+const PREBID_VERSION = '$prebid.version$';
+const ENDPOINT = 'https://c-testing.4dex.io/pba.gif';
```

Comment:
beware of this one

Reply:
Was going to forget about it

Reply:
Commit the change as soon as you can, we'll missed out :)